### DISCUSSION TITLE
Change default time chunk download

### DISCUSSION THREAD

**Discussion by jdavid22** [danielfernau/unifi-protect-video-downloader#96] [Ideas]
Hey,
I think it would be useful for users to change the download size from 1 hr to some user-requested duration. For example, when downloading 24 hours worth of footage instead of getting 24 files you could get less or more.

**Comment by danielfernau** [comment]
Hi there!
Thanks for the suggestion. The idea of using 1h long segments is based on the way the normal web GUI handles downloads. If you select, for example, 3 or 4 hours in the timeline and click on Download, you'll see that the native Protect GUI also splits the selection into 1h long files. This logic is in place to prevent the API from crashing, and to ensure that downloads can begin within a reasonable time after the initial request. 

However, adding an option for setting a custom clip length of less than 1h should be doable. There's already an issue that covers this request in its comments: https://github.com/danielfernau/unifi-protect-video-downloader/issues/34
I'll see what I can do :)
